ts/features/bonus/cgn/screens/merchants/CgnMerchantsTabsScreen.tsx

```diff
@@ -131,11 +131,14 @@ const CgnMerchantsTabsScreen: React.FunctionComponent<Props> = (
 
 const mapStateToProps = (_: GlobalState) => ({
   // FIXME replace with selector when available
-  merchants: availableMerchants
+  merchants: []
 });
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  navigateToMerchantDetail: () => dispatch(navigateToCgnMerchantDetail())
+  navigateToMerchantDetail: () =>
+    dispatch(
+      navigateToCgnMerchantDetail({ merchantID: "some" as Merchant["id"] })
```

Comment:
is it temporary?

Reply:
yes it is, the tab visualization is not actually connected at the moment since it is protected from the remote FF `cgn_merchants_v2`